- created a new model labeled Student.

BugaIulian opened the pull request:
-added all the parameters as private, e.g firstName, lastName etc 
-created a public constructor with null arguments.
-created a public constructor with all the parameters excluded only the id parameter.
 -created getters and setters all public for all the arguments except id. 
-some data type have been altered for example: cnp has now String datatype. 
-address has been branched into 5 parameters:
1. streetNumber
2. streetName
3. apartmentNumber
4. cityName
5. county
6. zipCode

 This can be altered once your pull request is resolved as there are some comments there and we need to decide if this is the way we are going with it.

BugaIulian committed:
- created a new model labeled Student.
-added all the parameters as private, e.g firstName, lastName etc
-created a public constructor with null arguments
-created a public constructor with all the parameters excluded only the id parameter.
-created getters and setters all public for all the arguments except id.
-some data type have been altered for example: cnp has now String datatype
-address has been branched into 5 parameters:
1. streetNumber
2. streetName
3. apartmentNumber
4. cityName
5. county
6. zipCode
This can be altered once your pull request is resolved as there are some comments there and we need to decide if this is the way we are going with it.

Miller1995 commented on src/main/java/com/startup/model/Student.java:
```diff
@@ -0,0 +1,112 @@
+    private String firstName;
+    private String lastName;
+    private String gender;
+    private String cnp;
```
If in class Teacher cnp is long, maybe you need use the same type of variable or it is need that change long in String in class Teacher ?
What do you think  ?

BugaIulian replied:
Do you want to use string as datatype? I think it's more easier as we already worked with strings an we know most of the string's specific methods! Also I think long it's usually used for big math operations so I don't think it will be proper to use it for cnp. Let me know what you think!

BugaIulian committed:
- created a new model labeled Student.
-added all the parameters as private, e.g firstName, lastName etc
-created a public constructor with null arguments
-created a public constructor with all the parameters excluded only the id parameter.
-created getters and setters all public for all the arguments except id.
-some data type have been altered for example: cnp has now String datatype
-address has been branched into 5 parameters:
1. streetNumber
2. streetName
3. apartmentNumber
4. cityName
5. county
6. zipCode
This can be altered once your pull request is resolved as there are some comments there and we need to decide if this is the way we are going with it.
